src/components/ItemForm/ItemForm.js

```diff
@@ -12,6 +12,8 @@ import {
   TextField,
 } from '@folio/stripes/components';
 import {
+  TitleManager,
+  withModules,
```

Comment:
@skomorokh, I pulled in `master` branch and locally I have a problem with `withModules` import.  Have you faced with similar problem?

Reply:
Nope, I'll give it a shot. One preliminary suggestion would be to `yarn why @folio/stripes-core` and see which you're using. Here's the commit from Jan 14th where that was exported: https://github.com/folio-org/stripes-core/commit/5aeaecfb8dd0492350a02654e3bee641c04b9d98

Reply:
Thanks for checking. Locally I do have the issue with version of `stripes-core`